static/js/public/snap-details/reportSnap.js

```diff
@@ -0,0 +1,89 @@
+import "whatwg-fetch";
+
+function toggleModal(modal, show) {
+  if (typeof show === "undefined") {
+    show = modal.style.display === "none";
```

Comment:
we should use `.u-hide` for toggling visibility

Reply:
Forgot completely about this utility. Thanks.